Userland/Applications/Spreadsheet/Tests/basic.js

```diff
@@ -139,3 +139,57 @@ describe("Range", () => {
         ).toEqual("<Cell at A0>,<Cell at A1>,<Cell at A2>,<Cell at B0>,<Cell at B1>,<Cell at B2>");
     });
 });
+
+describe("R function", () => {
+    const workbook = createWorkbook();
+    const sheet = createSheet(workbook, "Sheet 1");
+    sheet.makeCurrent();
+    /*
+      A B
+      +++
+    0+1 1
+    1+2 4
+    2+3 9
+    */
+    for (const row of ["A", "B"])
+        for (let col of [0, 1, 2]) sheet.setCell(row, col++, row === "A" ? col : Math.pow(col, 2));
+    sheet.focusCell("A", 0);
+    test("A", () => {
+        const range = R`A0:A`;
+        expect(range.toString()).toEqual("R`A0:A`");
+        expect(count(range)).toEqual(3);
+        expect(sum(range)).toEqual(6);
+    });
+    test("B", () => {
+        const range = R`A`;
+        expect(range.toString()).toEqual("R`A0:A`");
+        expect(count(range)).toEqual(3);
+        expect(sum(range)).toEqual(6);
+    });
+    test("C", () => {
+        const range = R`A0:B`;
+        expect(range.toString()).toEqual("R`A0:B`");
+        expect(count(range)).toEqual(6);
+        expect(sum(range)).toEqual(20);
+    });
+    test("D", () => {
+        const range = R`A0:B0`;
+        expect(range.toString()).toEqual("R`A0:B0`");
+        expect(count(range)).toEqual(2);
+        expect(sum(range)).toEqual(2);
+    });
+    // FIXME: This test should work but it only checks A0 and A2, not B0 and B2
+    // test('E', () => {
```

Comment:
I believe there's `test.skip(....)`, please use that instead of commenting it out

Reply:
I just ended it making it work :P